### ISSUE TITLE
Adobe Illustrator Beta has native Apple Silicon support

### ISSUE THREAD

**Issue by moffpage** [ThatGuySam/doesitarm#702]
**The official name of the app**
Adobe Illustrator

**Is there a supported version available on a stable release channel?**
No

**Proposed New App Status**
✅ Yes, Full Native Apple Silicon Support

**Related Issue Tracker Links or discussion**
https://www.reddit.com/r/AdobeIllustrator/comments/ld9xxr/apple_silicon_working/

**An Official App Download Page**
https://www.adobe.com/in/products/illustrator.html

**Additional context**
<img width="758" alt="image" src="https://user-images.githubusercontent.com/33549573/122328439-778d9480-cf51-11eb-858a-77bc986fe6dc.png">
<img width="690" alt="image" src="https://user-images.githubusercontent.com/33549573/122328452-7ceadf00-cf51-11eb-9c91-f5f80d98e619.png">
https://helpx.adobe.com/in/support/illustrator.html
https://helpx.adobe.com/in/illustrator/kb/illustrator-for-apple-silicon.html
<img width="1431" alt="image" src="https://user-images.githubusercontent.com/33549573/122328691-e5d25700-cf51-11eb-973c-20f9b922a6e4.png">
https://blog.adobe.com/en/publish/2021/06/08/illustrator-indesign-run-natively-apple-silicon-m1.html

**Full Screenshot with the App and Activity Monitor(if self reporting) or [Silicon Info](https://github.com/billycastelli/Silicon-Info)**
<img width="1270" alt="image" src="https://user-images.githubusercontent.com/33549573/122333694-58dfcb80-cf5a-11eb-95b4-6a0bf6f9f0a2.png">
<img width="377" alt="image" src="https://user-images.githubusercontent.com/33549573/122333691-58473500-cf5a-11eb-8663-fd49c9f9be5b.png">

**Comment by ThatGuySam** [comment]
Thanks for the report! 

For now, we'll wait on the stable release to update the listing so we're not communicating mixed signals.

**Comment by moffpage** [comment]
How does this update makes mixed signals?

**Comment by ThatGuySam** [comment]
People use this site to inform when they should purchase a new Mac so if we communicate that Illustrator is native too early we run the risk of leading someone to spend hundreds of dollars just to find out that the app they use to generate income(Illustrator) is only in beta for Native Apple Silicon support and the Beta isn't yet stable enough for their work. 

This could be made even worse if they sold their old Mac to purchase the new one. 

In another scenario, someone could use the information we provide to decide it's time to purchase a Mac for their Medical or Financial Institution. 

Sometimes it may not seem like what we do on this site has a big impact but it reaches way more industries than any of us realize.

**Comment by moffpage** [comment]
I don't get it why Illustrator specifically is exclusive to this kind of treatment from you. How's it different from other apps? It's always mentioned if the natively support version is part of a Beta / Early Access / Preview program. You could indicate that with another emoji if that strongly matters to you. isapplesiliconready.com does it this way, but for some reason I like your website more and like contributing to your project more, although I don't like the fact people have multiple data sources for this kind of thing.
Anyways, I don't think anybody should care about users heading into some issues because they've seen non-stable release of the app he's interested is available at the moment. He might've seen the similar news elsewhere, many companies do public betas (e.g. Apple, Google regular people (non-developers install iOS / iPadOS / macOS / Android betas on their main single devices or even brick their Apple Watches and then regret what they did - it's their own fault and you shouldn't feel sorry about them) etc etc. The point of the website is to give useful information for people to improve their workflow, prepare for changes and general contribution to this whole ARM-towards movement.
If a person isn't smart enough to firstly check if anything works properly (reading / watching reviews - even your website provides related videos if possibly, do testing / benchmarks of his workflow projects etc.) but rather relieve his own responsibilities (lol at selling his own Mac and buying a new one but then regretting - he can return it in 14 days globally and/or exchange it for an Intel model) to you or contributors - well, better tell this dummy to cry somewhere else.
I'm sure there aren't many frivolous people working at medical and financial institutions who'd rush to buy a new Apple Silicon based Mac without firstly being assured the very serious stuff they're working with actually works.
I agree with your last statement, but felt confused reading this comment from you. To recall - your website rather recommends to check the pricing for those new Macs, which someone may say alludes to purchasing an Apple Silicon-based Mac, rather than trying to restrain someone's attention from doing it.
Also, I don't like that Rider by JetBrains has green checkmark emoji, thus "Native Apple Silicon Support" (though "Initial" and "for EAP versions only"), but Android Studio "works via Rosetta 2 with native support in Preview". You didn't know Android Studio **Beta** has Apple Silicon support, not only Canary (aka "preview") versions? I'd like the info to be updated so there's the same writing about both apps. If you need me to initiate a new issue / update it, feel free to reply or contact in other ways.

**Comment by ThatGuySam** [comment]
What do you think the best solution is?

**Comment by moffpage** [comment]
You could do it like isapplesiliconready.com does with ⚠️ emoji. I think that fits nicely

**Comment by ThatGuySam** [comment]
So you're saying something like: 

> ⚠️ Yes, Native Apple Silicon Support

**Comment by moffpage** [comment]
Yes, but also a note that it’s available currently only in Beta / Preview / Early Access / etc. As you do now. And that should be for every app that has native non-stable Apple Silicon support.

**Comment by ThatGuySam** [comment]
Since we're trying to answer the question "Does It ARM?" or "Does this work on the new Macs?" could putting "⚠️" and then the word "Yes" be confusing to the average user?

**Comment by moffpage** [comment]
Someone could say that this warning emoji may frighten most users to download *probably better since it's optimized and usually delivers better performance, less memory usage etc. version*, even if it's in Beta / Early Access / Preview. Beta / Early Access / Preview doesn't always mean that the user gonna be less satisfied using that version. Thus, I think the details website should provide some highlighted information about the performance, memory usage, known issues, non-working features and so on. We aggregate some of this info - links to forums, Reddits and stuff, blogs, video reviews, benchmarks.
There's no issue if the warning emoji confuses some people, remember the purpose of this website - to present neutral data and let users decide what should they do with the information provided to them. Safe, but not off-putting. Users should get themselves either excited, suspicious, alerted, prepared by them own - it's their business.

**Comment by ThatGuySam** [comment]
Good news! Looks like Illustrator is now out of beta so we can go ahead and list it as the following: 
✅ Yes, Native Apple Silicon Support

**Comment by ThatGuySam** [comment]
Update is now live on the site! 

https://doesitarm.com/app/illustrator/

Feel free to comment on this issue for any future updates. 

Until then, you're welcome to browse through the other [M1 Issues](https://github.com/ThatGuySam/doesitarm/issues?q=is%3Aopen+is%3Aissue+label%3A%22Needs+M1+Testing%22) to help with missing info on other apps.
